src/aioftp/client.py

```diff
@@ -121,6 +123,7 @@ def __init__(
         parse_list_line_custom=None,
         parse_list_line_custom_first=True,
         passive_commands=("epsv", "pasv"),
+        explicit_tls=False,
```

Comment:
We do not need a parameter for this. Since this is a «explicit» tls command. So the user decide when to call `upgrade_to_tls`